src/coreComponents/constitutive/fluid/PVTFunctions/BrineEnthalpy.hpp

```diff
@@ -160,9 +160,7 @@ void BrineEnthalpyUpdate::compute( real64 const & pressure,
 
   if( useMass )
   {
-    // Do we really need to convert here from mole fraction to mass fraction?
-    real64 const X = C * waterMW / (C * waterMW + (1.0 - C) * CO2MW);
-    value = (1.0 - X ) * CO2Enthalpy + X * brineEnthalpy;
+    value = (1.0 - C ) * CO2Enthalpy + C * brineEnthalpy;
```

Comment:
After discussion with @francoishamon we decided that this conversion from molar fraction to mass fraction was a mistake and removed it in both old and new implementation.